<launch>
    <include file="$(find piper)/launch/start_single_piper.launch"/>

	<node pkg="sensor_tools" name="serial_gripper_imu" type="serial_gripper_imu" output="screen" respawn="true" />

    <include file="$(find pika_locator)/launch/pika_single_locator.launch"/>

     <node name="piper_IK" pkg="remote_operation" type="piper_ik.py" output="screen" >
        <rosparam command="load" file="$(find remote_operation)/config/piper_params.yaml" />
        <!-- Parameters gripper_xyzrpy and target_joint_state loaded from YAML -->
    </node>

</launch>
